frontend/src/employee-frontend/components/unit/unit-reservations/ChildDateModal.tsx

```diff
@@ -207,6 +213,24 @@ export default React.memo(function ChildDateModal({
   const reservationElems = useFormElems(reservations)
   const attendanceElems = useFormElems(attendances)
 
+  const expectedAbsencesRequest: ExpectedAbsencesRequest = useMemo(
+    () => ({
+      childId: child.id,
+      date,
+      attendances: attendances.isValid() ? attendances.value() : []
+    }),
+    [child.id, date, attendances]
+  )
+  const debouncedAbsencesRequest = useDebounce(expectedAbsencesRequest, 1500)
+  const expectedAbsencesEnabled =
+    date.isBefore(LocalDate.todayInHelsinkiTz()) &&
+    attendances.isValid() &&
+    !attendances.value().some((attendance) => attendance.endTime === null)
+  const expectedAbsences = useQueryResult(
+    childDateExpectedAbsencesQuery(debouncedAbsencesRequest),
+    { enabled: expectedAbsencesEnabled }
+  )
+
```

Comment:
This still has UX issues when playing around with values, like old cached warnings flashing after entering a new valid value.